src/IBM.WatsonDeveloperCloud/Http/Request.cs

```diff
@@ -109,6 +109,7 @@ public TaskAwaiter<IResponse> GetAwaiter()
         public async Task<T> As<T>()
         {
             HttpResponseMessage message = await this.AsMessage().ConfigureAwait(false);
+            var result = message.Content.ReadAsStringAsync().Result;

```

Comment:
Currently in the examples I am tracing out the major components of the returned object. German and Jeff said in the examples we should just print the returned json. It looks like in the Java SDK it returns an object but the Python SDK returns json.